modules/core/src/v2/coins/xlm.ts

```diff
@@ -110,10 +119,25 @@ interface VerifyAddressOptions extends BaseVerifyAddressOptions {
   rootAddress: string;
 }
 
+interface TrustlineOptions {
+  token: string;
+  action: string;
+  limit?: string;
+}
+
+interface TransactionParams extends BaseTransactionParams {
+  trustlines?: TrustlineOptions[];
+}
+
+interface VerifyTransactionOptions extends BaseVerifyTransactionOptions {
+  txParams: TransactionParams;
+}
+
 export class Xlm extends BaseCoin {
   public readonly homeDomain: string;
   public static readonly tokenPatternSeparator = '-'; // separator for token code and issuer
   static readonly maxMemoId: string = '0xFFFFFFFFFFFFFFFF'; // max unsigned 64-bit number = 18446744073709551615
+  static readonly maxTrustlineLimit: string = '922337203685.4775807'; // max int64 number supported by the network ((2^63)-1)/(10^7)
```

Comment:
Nit: consider leaving a link to https://www.stellar.org/developers/guides/concepts/assets.html#amount-precision-and-representation so the next person doesn't find this magic number so magical

Reply:
ok, done